<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="labelUser">
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="5">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>get</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelUser_2">
     <property name="text">
      <string>服务器ip：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditIP">
     <property name="text">
      <string>42.194.195.184</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelPwd">
     <property name="text">
      <string>密 码：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>服务器的回复：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditUser">
     <property name="text">
      <string>测试</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditPwd">
     <property name="text">
      <string>123456</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelUser_3">
     <property name="text">
      <string>服务器端口：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditPort">
     <property name="text">
      <string>80</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
